src/UI/CredentialsAutofill.js

```diff
@@ -0,0 +1,53 @@
+const {
```

Comment:
I've abstracted common logic to a `Tooltip` class, while these implementation add specific stuff like markup, styles and listeners.